lms/static/js/student_account/views/AccessView.js

```diff
@@ -56,10 +56,7 @@
                     this.accountActivationMessages = options.account_activation_messages || [];
 
                     if (options.login_redirect_url) {
-                    // Ensure that the next URL is internal for security reasons
-                        if (! window.isExternal(options.login_redirect_url)) {
```

Comment:
Question: Can you explain this change?

Reply:
We need to remove the `window.isExternal` check to allow for cross domain redirect URLs.